### ISSUE TITLE
Did Option addition/subtraction/etc ever make it into the package?

### ISSUE THREAD

**Issue by MrYossu** [louthy/language-ext#976]
According to a comment by @louthy in the last post in [this issue](https://github.com/louthy/language-ext/issues/33#issuecomment-148791681), we should be able to do this...

```c#
var s1 = Some(10);
var s2 = Some(20);
var s3 = s1 + s2;   // Some(30)
```

That was posted back in Oct '15. I just tried this (using v4.0.3) and got a compiler error on the addition, _CS0019 Operator '+' cannot be applied to operands of type `Option<int>` and `Option<int>`_.

Did I mis something, or is this not supported?

Thanks

**Comment by louthy** [comment]
No, it's not possible as the `operator+` declaration must be part of the `Option` struct, which means it's working with an unconstrained generic type, not an `int`.
